frontend/src/App.tsx

```diff
@@ -0,0 +1,102 @@
+import { useQuery, gql, DocumentNode } from "@apollo/client";
+import React from "react";
+import { theme } from "@diamondlightsource/ui-components"
+import { ChakraProvider } from "@chakra-ui/react";
+import { LoadMoreTable } from "./components/LoadMoreTable";
+
+const GET_INFO: DocumentNode = gql`
+query pinInfo ($after: String) {
+  libraryPins(first: 2, after: $after) {
+    pageInfo {
+      hasPreviousPage,
+      hasNextPage,
+      startCursor,
+      endCursor
+    },
+    edges {
+      cursor
+      node {
+        barcode,
+        loopSize,
+        status
+      }
+    }
+  }
+}
+`;
+
+// Displays libraryPins query in table component. The table can load more data if required 
+function DisplayPinInfo(): React.JSX.Element {
+  const { loading, error, data, fetchMore } = useQuery(
+    GET_INFO,
+    {
+      notifyOnNetworkStatusChange: true,
+    });
+
+  if (error) return <p>Error : {error.message} {error.extraInfo}</p>;
```

Comment:
Is there a better way we could deal with errors? Perhaps an [alert](https://chakra-ui.com/docs/components/alert)